### ISSUE TITLE
Strawberry skull

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7371]
**Preferred term label:**
Strawberry skull

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormal skull shape ascertained by prenatal imaging that is said to resemble a strawberry.

PMID:30363185
PMID: 26498670

Question -- is this the same as fetal trigonocephaly and do we need a new term?

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by marziapollazzon** [comment]
I think that we don't need a new term. In the recent PMID: 33769585 already in the title there are the two synonimous terms already in the title "Fetal Trigonocephaly (Strawberry Skull)". The definition I propose is "flattening of the occiput with pointing of the frontal bones", as reported in the article.

**Comment by IsabelFilges** [comment]
Synonym is also strawberry-shaped skull which is an ultrasound marker defined by
- flattening of the occiput with pointing of the frontal bones and 
- brachycephaly with an increased cephalic index
I am not sure it is equivalent to trigonocephaly only, so probably does not need an additional term, but  I think it should somehow be represented in the hierarchical relationship
Strawberry skull can be seen in trisomy 18 (or other chromosomal abnormalities) and monogenic syndromes, particularly also in  craniosynostosis syndromes and some skeletal dysplasias

**Comment by carlotarodo** [comment]
Despite this term is "classical" I think we should not use "fruits" to define morphologic features

**Comment by pnrobinson** [comment]
@carlotarodo @IsabelFilges is there an alternative name for this then?

**Comment by IsabelFilges** [comment]
I am not a sonographer or MFM speciliast, but a clinical geneticist - but for the sonographer the "fruit" descriptions are well defined terms according to what they assess/see as a phenotype. The significance of those terms (association with cooresponding genetic conditions) are, to my knowledge, well known also to the geneticists working in that field. For practical reasons I would advocate to keep those descriptive terms commonly used in clinical practice. I think we also know such terms from other (postnatal) clinical fields, e.g. raspberry tongue in scarlet fever. Every physician knows what it means and associates a diagnosis with it.

**Comment by carlotarodo** [comment]
Despite @IsabelFilges comments are true, I still think we could try to describe the images properly. I would agree with @marziapollazzon: "flattening of the occiput with pointing of the frontal bones" or "trigonocephaly".

**Comment by umberagarwal** [comment]
This is same as fetal trigonocephaly. It has also been called as 'Mozart's skull' in literature by mistake(https://doi.org/10.1002/ajpa.1330940209).

The term trigonocephaly was coined by Welcker in 1862

I like this description better :  **It is characterized by keel shaped deformity of forehead with midline ridge, bilateral fronto temporal constriction with compensatory biparietal expansion, supra orbital and lateral orbital retrusion and hypotelorism.** (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4166832/

There are 3 entries in OMIM for trigonocephaly- #614485, #190440, and 314320

https://orcid.org/my-orcid?orcid=0000-0001-7600-5359

**Comment by pnrobinson** [comment]
Let's add this term with primary label "Fetal trigonocephaly" and synonym "Strawberry etc" so that people who are using the "old" terminology can find it.

@sng2140
Added as HP:4000142
